app/src/main/java/com/simplemobiletools/calculator/Utils.kt

```diff
@@ -0,0 +1,10 @@
+package com.simplemobiletools.calculator
+
+import android.content.Context
+import android.widget.Toast
+
+object Utils {
+    fun showToast(context: Context, resId: Int) {
+        Toast.makeText(context, context.resources.getString(resId), Toast.LENGTH_SHORT).show()
+    }
```

Comment:
reuse the toast extension from Commons library, you can then remove this whole file